label_sync/BUILD

```diff
@@ -2,12 +2,47 @@ package(default_visibility = ["//visibility:public"])
 
 licenses(["notice"])
 
+# Usage:
+#   bazel run //label_sync:image  # build image
+#   bazel run //label_sync:push  # push image (after building)
+
 load(
     "@io_bazel_rules_go//go:def.bzl",
     "go_binary",
     "go_library",
     "go_test",
 )
+load(
+    "@io_bazel_rules_docker//docker:docker.bzl",
+    "docker_build",
+    "docker_bundle",
+)
+load(
+    "@io_bazel_rules_docker//contrib:push-all.bzl",
+    "docker_push",
+)
+
+docker_build(
```

Comment:
Consider changing this to a `go_image` rule.

Reply:
Thanks! Docs on the go_image rule?

Reply:
[Here's](https://github.com/kubernetes/test-infra/pull/4929#pullrequestreview-80041244) my PR adding it elsewhere in Prow, which may be most directly useful.  [Here](https://github.com/bazelbuild/rules_docker#language-rules) are the docs for `lang_image` rules.

Reply:
I want to punt this to a subsequent PR